external/fv3fit/tests/emulation/layers/test_normalization.py

```diff
@@ -90,7 +89,4 @@ def test_NormLayer_Configs(config_cls):
 
     tensor = _get_tensor()
     config = config_cls("mean_std", tensor, "lol")
-    layer = config.initialize_layer()
-
-    assert isinstance(layer, NormLayer)
```

Comment:
these kinds of checks are pretty rigid esp if `NormLayer` is a concrete class rather than a protocol.